cmake/modules/AddSwift.cmake

```diff
@@ -1387,15 +1390,62 @@ endfunction()
 #   Sources to add into this library.
 function(add_swift_library name)
   set(SWIFTLIB_options
-      SHARED STATIC OBJECT_LIBRARY IS_STDLIB IS_STDLIB_CORE IS_SDK_OVERLAY
-      TARGET_LIBRARY FORCE_BUILD_FOR_HOST_SDK
-      API_NOTES_NON_OVERLAY DONT_EMBED_BITCODE HAS_SWIFT_CONTENT
-      FORCE_BUILD_OPTIMIZED NOSWIFTRT)
+        API_NOTES_NON_OVERLAY
```

Comment:
I don't think we should use this PR as a means to re-format these arguments. I'd much prefer if we kept whitespace changes in separate, non-functional PRs if possible. Makes it not only easier to review, but also distills PRs such as this one down to just the functional changes.